sources/packages/backend/apps/api/src/services/education-program-offering/education-program-offering.service.ts

```diff
@@ -1060,8 +1067,10 @@ export class EducationProgramOfferingService extends RecordDataModelService<Educ
         "precedingOffering.id",
         "location.id",
         "institution.id",
+        "educationProgram",
```

Comment:
Please do not select the entire object when it is not required. 

![image](https://github.com/bcgov/SIMS/assets/54600590/c56894e2-c33f-4a4d-8f33-10187d8757da)